api/docs/api.yml

```diff
@@ -1,1511 +1,1579 @@
-openapi: 3.0.0
```

Comment:
So.. erm.. what changed here? :joy:

Reply:
Ooh, the diff was a lot easier to make sense of in VS code! Basically, I changed the query param, request body schema and response for the publication create endpoint to reflect the changes in this PR.